### PR TITLE
Review Style conventions page

### PR DESCRIPTION
**Pull request by jdbaldry**
- Use contractions
- Use semantic line breaks
- Update admonition syntax

Closes https://github.com/grafana/writers-toolkit/issues/625

### CODE REVIEW

File: docs/sources/write/style-guide/style-conventions/index.md [hunk 1]
```diff
@@ -214,17 +226,19 @@ A caution emphasizes a course of action's potential downsides.
 The syntax for a caution admonition is as follows:
 
 ```markdown
-{{%/* admonition type="caution" */%}}
+{{</* admonition type="caution" */>}}
 By disabling authentication requirements, anyone can access your Grafana instance.
 There is a considerable security risk associated with this.
-{{%/* /admonition */%}}
+{{</* /admonition */>
+}}
```
Comment (clayton-cornell) [inline docs/sources/write/style-guide/style-conventions/index.md:233]:
```suggestion
{{</* /admonition */>}}
```

File: docs/sources/write/style-guide/style-conventions/index.md [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
+title: Style conventions
+weight: 200
 ---
 
 # Style conventions
 
-This section includes a non-exhaustive list of technical writing techniques and styles to consider when you write technical content. For questions you might have that are not addressed in these guidelines topic, refer to
-[Google developer documentation style guide](https://developers.google.com/style).
+Consider this non-exhaustive list of technical writing techniques and styles when you write technical documentation.
+For questions you might have that aren't addressed in these guidelines topic, refer to [Google developer documentation style guide](https://developers.google.com/style).
```
Comment (clayton-cornell) [inline docs/sources/write/style-guide/style-conventions/index.md:16]:
```suggestion
For questions you might have that aren't addressed in these guidelines topic, refer to the [Google developer documentation style guide](https://developers.google.com/style).
```

Comment (josmperez) [reply docs/sources/write/style-guide/style-conventions/index.md:16]: delete word "topic"

File: docs/sources/write/style-guide/style-conventions/index.md [hunk 3]
```diff
@@ -293,15 +307,16 @@ To add an administrator to the list of local users, navigate to **Appliance** >
 
 ### Italic
 
-Use italics formatting (\_ in Markdown), to emphasize a specific word or phrase. This is particularly useful when defining a term for the first time.
+Use italics formatting (`_`), to emphasize a specific word or phrase.
```
Comment (josmperez) [inline docs/sources/write/style-guide/style-conventions/index.md:310]: I think it's "Use italics" or "Use italic formatting"

**Review by josmperez — APPROVED** [review]
Overall LGTM.

**Pull request merged**